webpack/test_setup.js

```diff
@@ -1,9 +1,11 @@
+import 'babel-polyfill';
```

Comment:
Need to add `babel-polyfill` so tests will run on `nodejs < 8`

`Object.values` and `Object.entries` are introduced in `nodejs@8`